index.html

```diff
@@ -379,149 +379,64 @@ <h2>Former Members</h2>
-</div>
-<div class="timeline-body">
-<p class="text-muted">
-PSB agrees to host the Research Parasite Awards, which will be given recipients determined
-by an award committee annually at the meeting.
-</p>
-</div>
-</div>
-</li>
-<li class="timeline-inverted">
-<div class="timeline-image">
-<br><h4>The Prizes</h4>
-</div>
-<div class="timeline-panel">
-<div class="timeline-heading">
-<h4>May - August 2016</h4>
-<h4 class="subheading">Supporters offer prizes.</h4>
-</div>
-<div class="timeline-body">
-<p class="text-muted">A series of supporters, including journals, foundations, and individuals
-who wish to promote the rigorous and reproducible analysis of existing data offer support to
-the award, allowing a cash prize and travel award to be established.</p>
-</div>
-</div>
-</li>
-<li class="timeline">
-<div class="timeline-image">
-<h4>Be Part
-<br>Of The
-<br>Awards!</h4>
-</div>
-<div class="timeline-panel">
-<div class="timeline-heading">
-<h4>January 2017</h4>
-<h4 class="subheading">First awards to be given at PSB</h4>
-</div>
-<div class="timeline-body">
-<p class="text-muted">In January of 2017, the first Research Parasite Awards will be given
-for junior and sustained parasitism. If you have a contribution that meets the requirements
-please <a class="page-scroll" href="#apply">apply </a>. If you'd like to work with us to
-recognize these contributions, please <a class="page-scroll" href="#prize">get in touch</a>.</p>
-</div>
-</div>
-</li>
-</ul>
-</div>
-</div>
-</div>
-</section>
--->
-
 <!-- Past Recipients Grid Section -->
-<!--
 <section id="past-recipients">
-<div class="container">
-<div class="row">
-<div class="col-lg-12 text-center">
-<h2 class="section-heading">Award Recipients</h2>
-<h3 class="section-subheading text-muted">Exemplars of research parasitism.</h3>
-</div>
-</div>
-<div class="row">
-<div class="col-md-6 col-sm-6 portfolio-item">
-<a href="#KunHsingYuModal" class="portfolio-link" data-toggle="modal">
-<div class="team-member">
-<img src="img/past-recipients/Kun-Hsing_Yu.png" class="img-responsive img-circle" alt="">
-<h4>Kun-Hsing Yu</h4>
-<p class="text-muted">2017 Junior Parasite</p>
-</div>
-</a>
-</div>
-<div class="col-md-6 col-sm-6 portfolio-item">
-<a href="#ErickTurnerModal" class="portfolio-link" data-toggle="modal">
-<div class="team-member">
-<img src="img/past-recipients/Erick_Turner.png" class="img-responsive img-circle" alt="">
-<h4>Erick Turner</h4>
-<p class="text-muted">2017 Sustained Parasitism</p>
-</div>
-</a>
-</div>
-</div>
-</div>
+ <div class="container">
+  <div class="row">
+   <div class="col-lg-12 text-center">
+    <h2 class="section-heading">Award Recipients</h2>
+    <h3 class="section-subheading text-muted">Exemplars of research symbiosis.</h3>
+   </div>
+  </div>
+
+  <div class="row">
+   <div class="col-md-6 col-sm-6 portfolio-item">
+    <a href="#LeonardoColladoTorresModal" class="portfolio-link" data-toggle="modal">
```

Comment:
I think you want to get rid of these `<a>` elements because you don't have the modal that shows up (check further down the page, i think some are commented out. You can also snag the code from the parasite site if you prefer.

Reply:
done, skipping the modals for now for time reasons